### ISSUE TITLE
Naive run-through of README failed

### ISSUE THREAD

**Issue by mrocklin** [dask/dask-kubernetes#4]
So I ran through the README got this point and then failed with the following error:

```
mrocklin@workstation:~/workspace/dask-kubernetes$ source scripts/make_cluster.sh 
Updated property [compute/zone].
ERROR: (gcloud.container.clusters.create) ResponseError: code=404, message=The resource "projects/dask-cluster" was not found.
Fetching cluster endpoint and auth data.
ERROR: (gcloud.container.clusters.get-credentials) ResponseError: code=404, message=The resource "projects/dask-cluster" was not found.
Unable to connect to the server: dial tcp 104.196.158.131:443: i/o timeout
```

My first guess here is that I have some config file hidden somewhere on my computer pointing gcloud to use an old resource.  I suspect that I'm failing to properly execute this step in the README:

> Register on the [Google Cloud Platform](https://cloud.google.com/), setup a billing account and create a project with the Google Compute Engine API enabled.

I am (perhaps naively) assuming that my previous projects on my work account already satisfy this constraint.  Looking on there now it looks like there are a couple of these active.  How does gcloud choose between them?  How does it choose between these and my personal account?  Do we know why it is finding this old `dask-cluster` project?

This is probably all on me rather than on this project, but any thoughts on how gcloud determines what project is uses would be welcome.

**Comment by martindurant** [comment]
gcloud typically (always?) stores things in `~/.config/gcloud/`. There are such things as [named configurations](https://cloud.google.com/sdk/gcloud/reference/topic/configurations) to switch between identities, but I'm not sure how this works, aside from reading the linked page.

**Comment by mrocklin** [comment]
OK, that was helpful.  Calling `gcloud init` walked me through the choices necessary to select the correct account and project.  Depending on how other people react to this situation it may be useful to include a note including a pointer to this operation.

**Comment by mrocklin** [comment]
OK, next problem.  This repeatedly shows no results:

```
mrocklin@workstation:~/workspace/dask-kubernetes$ kubectl get pods -l app=distributed
No resources found.
```

This took a few minutes so I logged onto the console, clicked on my google container engine and looked at my application.  That gave me instructions to start a kubernetes proxy locally.  This shows that things do appear to be running.  I was able to click on the links there to see that the scheduler and jupyter notebook are both running.  

Is there perhaps something missing from the README instructions about connecting my local kubernetes commands to the cluster running on google cloud?

**Comment by mrocklin** [comment]
It would be good to state the default jupyter notebook server password in the readme.  I think that the default used by jupyter is `jupyter`.  We should possibly use this.

**Comment by mrocklin** [comment]
It would be useful to expose port 8789 from each of the workers, if possible

**Comment by martindurant** [comment]
"It would be useful to expose port 8789"
This isn't possible, or at least not simple. The way to expose a port from a set of pods is to use a load-balancer, but the whole point of this is that you get the one port, and the request will be served by one of the pods, you don't know which. I haven't see the case where you get a port from each of a set of pods.

"the default used by jupyter is jupyter"
I don't think there is a default, you simply can't access the notebooks without an explicitly set password, except on localhost

" repeatedly shows no results"
Sounds like the labels failed, for some reason. The pods presumably show up without the -l option - I wonder what labels they got? It should take no time at all for this to show results, only those results will not have "ready" status for a while.

**Comment by martindurant** [comment]
It seems labels were applied to services and resource-controlers, but not pods (odd, because it worked before). I fixed it #3 .
